src/pages/HomePage/HomePage.tsx

```diff
@@ -94,15 +109,17 @@ const renderLastSearchOptions = (): JSX.Element[] => {
 };
 
```

Comment:
use useCallback for renderLastSearchOptions and useMemo for filteredLastSearches